frame/referenda/src/lib.rs

```diff
@@ -579,6 +595,36 @@ pub mod pallet {
 				};
 			Ok(Some(branch.weight::<T, I>()).into())
 		}
+
+		/// Refund the Submission Deposit for a closed referendum back to the depositor.
+		///
+		/// - `origin`: must be `Signed` or `Root`.
+		/// - `index`: The index of a closed referendum whose Submission Deposit has not yet been
+		///   refunded.
+		///
+		/// Emits `SubmissionDepositRefunded`.
+		#[pallet::weight(T::WeightInfo::refund_submission_deposit())]
+		pub fn refund_submission_deposit(
+			origin: OriginFor<T>,
+			index: ReferendumIndex,
+		) -> DispatchResult {
+			ensure_signed_or_root(origin)?;
```

Comment:
using some newlines could make your code generally a bit more easy on the eyes.

Reply:
controversial topic, like spaces vs tabs.
I prefer no new lines for example, it makes me not jump between paragraphs, and read line by line. 
So functions only as a separation of concept.